lib/Migration/Version000102Date20191004200147.php

```diff
@@ -81,7 +81,7 @@ public function changeSchema(IOutput $output, Closure $schemaClosure, array $opt
 			]);
 			$table->addColumn('credential_public_key', 'string', [
 				'notnull' => true,
-				'length' => 255
+				'length' => 2000,
```

Comment:
If you change this you might want to add another migration to make sure existing installs have the same value, right?

Reply:
https://github.com/nextcloud/twofactor_webauthn/blob/main/lib/Migration/Version000200Date20200310200500.php#L48 does that. I've changed the migration that adds the column so widening of the column is only necessary of existing installations but means a NOOP for new ones.